livedata/src/main/java/io/getstream/chat/android/livedata/service/sync/SyncService.kt

```diff
@@ -7,43 +7,48 @@ import android.content.Context
 import android.content.Intent
 import android.os.Build
 import android.os.IBinder
+import android.util.Log
 import androidx.core.app.NotificationCompat
+import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.events.NewMessageEvent
+import io.getstream.chat.android.client.models.User
 import io.getstream.chat.android.livedata.ChatDomain
 
 class SyncService : Service() {
 
     override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
         createNotificationChannel()
+        showForegroundNotification()
 
+        intent?.getStringExtra(EXTRA_CID)?.let {
+            val config = EncryptedBackgroundSyncConfigStore(applicationContext).get()
```

Comment:
What happen if the returned config is `BackgroundSyncConfig.UNAVAILABLE`?
Shouldn't we stop the sinchronization process?